content/kapp-controller/docs/latest/packaging.md

```diff
@@ -0,0 +1,219 @@
+  # App template used to create the underlying app. See 'App CR Spec' docs for more
+  # info
+  template:
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: registry.tkg.vmware.run/tkg-fluent-bit@sha256:...
+      template:
+      - ytt:
+          paths:
+          - config
+      - kbld:
+          paths:
+          - -
+          - .imgpkg/images.yml
+      deploy:
+      - kapp: {}
+```
+
+### PackageRepository CR
+
+This CR is used to point kapp-controller to a store of Package CRs. Once a
+PackageRepository has been added to the cluster, kapp-controller will automatically
+make any packages within the store available for installation on the cluster.
+
+PackageRepository Spec:
+
+```yaml
+apiVersion: install.package.carvel.dev/v1alpha1
+kind: PackageRepository
+metadata:
+  name: basic.vmware.com
+  # cluster scoped
+spec:
+  # Must have only one directive.
+  fetch:
+    # pulls content from imgpkg bundle stored in Docker/OCI registry
+    imgpkgBundle:
+      # Image url; unqualified, tagged, or
+      # digest references supported (required)
+      image: host.com/username/image:v0.1.0
+      # secret with auth details (optional)
+      secretRef:
+        name: secret-name
+    # pulls image containing packages from Docker/OCI registry
+    image:
+      # Image url; unqualified, tagged, or
+      # digest references supported (required)
+      url: host.com/username/image:v0.1.0
+      # secret with auth details (optional)
+      secretRef:
+        name: secret-name
+      # grab only portion of image (optional)
+      subPath: inside-dir/dir2
+    # uses http library to fetch file containing packages
+    http:
+        # http and https url are supported;
+        # plain file, tgz and tar types are supported (required)
+        url: https://host.com/archive.tgz
+        # checksum to verify after download (optional)
+        sha256: 0a12cdef83...
+        # secret to provide auth details (optional)
+        secretRef:
+          name: secret-name
+        # grab only portion of download (optional)
+        subPath: inside-dir/dir2
+    # uses git to clone repository containing package list
+    git:
+        # http or ssh urls are supported (required)
+        url: https://github.com/k14s/k8s-simple-app-example
+        # branch, tag, commit; origin is the name of the remote (required)
+        ref: origin/develop
+        # secret with auth details. allowed keys: ssh-privatekey, ssh-knownhosts, username, password (optional)
+        # (if ssh-knownhosts is not specified, git will not perform strict host checking)
+        secretRef:
+          name: secret-name
+        # grab only portion of repository (optional)
+        subPath: config-step-2-template
+        # skip lfs download (optional)
+        lfsSkipSmudge: true
+```
+
+
+
+### InstalledPackage
+
+This CR is used to inform kapp-controller that a user desires an installed
+package. Once kapp-controller notices this desired state, it will work to make
```

Comment:
```suggestion
This CR is used to inform kapp-controller that a user wants to install a
package. Once kapp-controller notices this desired state, it will work to make
```